pkg/util/kernel/version.go

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
`//+build linux`

Reply:
@thockin 
```shell
ERROR(windows/386): 
/home/prow/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/proxy/ipvs/proxier.go:55:13: 
could  not import k8s.io/kubernetes/pkg/util/kernel (invalid package name: "")
```

I guess we need to add build constraints to all the consumers of this pkg as well, should I skip it ?

Reply:
everything in `pkg/proxy/iptables` and `pkg/proxy/ipvs` really ought to be `//+build linux`, though that won't currently work, because our CI currently requires kube-proxy to be buildable on OS X even though the binary won't actually work there (#118020). So before we can make `pkg/proxy/ipvs` be linux-only, we have to either fix CI or make `cmd/kube-proxy` have a dummy OS X backend that just immediately errors out.

Which is getting a bit far afield for this issue so maybe just file a bug reminding us to revisit it?

Reply:
ACK

Reply:
i removed the build constraints and will file an issue for the same.

Reply:
I filed #121722 to try to fix this...